Lab5/src/common/types/user/user.properties.ts

```diff
@@ -0,0 +1,5 @@
+import {UserEntity} from "../../../entity/user/user.entity.js";
```

Comment:
../../../ usually is a mark of wrong reference. Better use reference from root, or add ts path aliases